pkg/clusteragent/autoscaling/workload/model/pod_autoscaler.go

```diff
@@ -484,3 +561,41 @@ func newCondition(status corev1.ConditionStatus, reason string, currentTime meta
 
 	return condition
 }
+
+func getHorizontalEventsRetention(policy *datadoghq.DatadogPodAutoscalerPolicy, longestLookbackAllowed time.Duration) time.Duration {
+	var longestRetention time.Duration
+	if policy == nil {
+		return longestRetention
```

Comment:
```suggestion
		return 0
```
IMO we should be explicit here